pkg/controller/bypass/bypass_controller.go

```diff
@@ -78,15 +80,25 @@ func StartByPassController(client kubernetes.Interface) error {
 
 			log.Infof("%s/%s: ADDED", pod.GetNamespace(), pod.GetName())
 			enableSidecar, _ := checkSidecar(client, pod)
-			if !enableSidecar {
-				log.Info("do not need add iptables rules, pod is not managed by sidecar")
+			enableKmesh := checkKmesh(pod)
+			if !enableSidecar && !enableKmesh {
```

Comment:
we can remove this check now

Reply:
> we can remove this check now

Yes, the nspath variable needs to be reused in the earlier version. The obtaining of this variable is time-consuming. Now, the nspath variable needs to be obtained in both scenarios. Therefore, I have removed this check.